test/bot_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from app.connect_db import add_new_user, get_all_users, delete_user, get_language_from, get_language_target, \
+                            update_language_target_for_user, update_language_from_for_user
+
+test_user_id = 125
+
+
+@pytest.fixture(autouse=True)
+def create_user():
+    add_new_user(test_user_id)
+    yield
+    delete_user(test_user_id)
+
+
+def test_add_new_user_without_lang():
+    assert test_user_id in get_all_users()
+
+
+def test_add_new_user_with_lang():
+    test_user_id = 123
+    delete_user(test_user_id)
+    lang_target = "fr"
+    lang_from = "en"
+    add_new_user(test_user_id, lang_target, lang_from)
```

Comment:
a bit more concise without readability loss
```suggestion
    add_new_user(test_user_id, lang_target="fr", lang_from="en")
```

Reply:
I still use these variables in assert

Reply:
ok then. though in tests it would be fine to use constant strings in assert as well.